components/ui/browser-frame.module.css

```diff
@@ -0,0 +1,5 @@
+@media (prefers-color-scheme: dark) {
```

Comment:
this will break if we allow ppl to switch between `dark/light/system` themes... 

making it a client-component that uses `useTheme` from next-themes is causing a flash 😭

Reply:
https://github.com/pacocoursey/next-themes?tab=readme-ov-file#css there's this solution but that would cause two instances of the demo 🤡